elide-core/src/main/java/com/yahoo/elide/core/DataStoreTransaction.java

```diff
@@ -31,7 +32,7 @@
         PARTIAL,
         NONE
     }
-
+    UUID TRANSACTION_ID = UUID.randomUUID();
```

Comment:
Won't this be static?  This doesn't look right to me.

Reply:
Don't we want different **TRANSACTION_ID** for each transaction? And making it static would mean 1 id for all the instances of the class?